components/organisms/o-shipping.vue

```diff
@@ -47,7 +47,7 @@
         :error-message="$t('Field is required')"
         @blur="$v.shipping.streetAddress.$touch()"
       />
-      <SfInput
+      <!-- <SfInput
```

Comment:
we dont want to hide this, we want to make it not-required.

Reply:
we don’t currently save apt number anywhere though. Would you want to add it onto the end of street or something?

Reply:
oh, is that so? in that case, then yes. removing it makes sense OR append it to our street. thoughts? i think its very common to have this field.

Reply:
Attaching the house number to the street would be an extremely easy option to do.

Reply:
OK fine, but please open an issue to have a second address line in apicur.io.

Reply:
and after you do, please add a link to the issue here to the comment (in the code).

Reply:
So... you mean something like this?
![image](https://user-images.githubusercontent.com/54149336/116943915-1a0be580-ac3a-11eb-80b1-17239d03e402.png)

Reply:
I think it's done.